assets/js/phoenix_live_view.js

```diff
@@ -407,7 +407,8 @@ export class LiveSocket {
   }
 
   bindTopLevelEvents(){
-    this.bindClicks()
+    const clickListener = ("ontouchstart" in window) ? "touchstart" : "click"
```

Comment:
a few minor concerns with this approach that I'm not sure if totally valid or not.  Given multi input devices and the 300ms artificial delay to `touch` events, this may be a tad brittle or troll implementors/developers. 

I think this does need to be fixed.  Just not sure of a specific solution off the top of my head.

Reply:
You are right, it is not a fix for every corner case. 

I saw a more complete solution here https://stackoverflow.com/a/13358390, which better limits the `tap` event to be "a `touchend` that hasn't `touchmove`d after a `touchstart`". My change took a simpler approach, because 1) it works for me, 2) it just adds _some_ support in iOS without breaking any existing features, 3) it makes the intention clear.

Reply:
Okay, I made that change, and it worked. I will close this PR, and make a new one.